File: Beecrowd/Beginner/1101 - 1190/1164 - Perfect Number/C/1164_PerfectNumber.c
#include <stdio.h>

int main() {
    int testCases;
    scanf("%d", &testCases);

    for (int i = 0; i < testCases; i++) {
        int number, sum = 0;
        scanf("%d", &number);

        for (int j = 1; j < number; j++) {
            if (number % j == 0) {
                sum += j;
            }
        }

        if (sum == number) {
            printf("%d eh perfeito\n", number);
        } else printf("%d nao eh perfeito\n", number);
    }

    return 0;
}

File: Beecrowd/1 - Beginner/1101 - 1190/1164 - Perfect Number/1164_PerfectNumber.py
test_cases = int(input())

for _ in range(test_cases):
    number = int(input())
    sum_divisors = 0

    for j in range(1, number):
        if number % j == 0:
            sum_divisors += j

    if sum_divisors == number:
        print(f"{number} eh perfeito")
    else:
        print(f"{number} nao eh perfeito")
